frontend/src/components/TestedPerDayPositivePercentLineChart.svelte

```diff
@@ -0,0 +1,71 @@
+<script lang="ts">
+  import c3, { ChartConfiguration, ChartAPI, PrimitiveArray } from "c3";
+
+  import {
+    formatNumber,
+    getChartConfig,
+    getTestedPerDayPercentChartRows,
+  } from "../utils";
+  import { infectedIncreaseLog } from "../store";
+  import ChartLayout from "./ChartLayout.svelte";
+
+  export let title: string;
+  export let id: string;
+
+  let timespan = 14;
+  let rows: PrimitiveArray[] = [];
+  let chart: ChartAPI | undefined;
+
+  function tooltipValue(
+    value: number,
+    ratio: number | undefined,
+    id: string,
+    index: number
+  ): string {
+      (rows[index + 1][1] as number) + (rows[index + 1][2] as number);
+    const percentage = Math.round(value * 10000) / 100;
+    return `${percentage}%)`;
+  }
+
+  function labelsFormatter(value: number): string {
+    if (timespan === -1) {
+      return "";
+    }
+    return formatNumber(value);
```

Comment:
formatNumber just add spaces after each 3 characters. Quite misleading. But rounding is required for this value.

Reply:
Added rounding